scripts/release/upload_metadata_file.py

```diff
@@ -0,0 +1,83 @@
+import argparse
```

Comment:
nit: Should we add typing? I know it's a pain but it can be helpful for longer term maintenance

Reply:
Added